sdk/playwrighttesting/create-microsoft-playwright-testing/README.md

```diff
@@ -0,0 +1,54 @@
+# Microsoft Playwright Testing preview
+
+Microsoft Playwright Testing is a fully managed service that uses the cloud to enable you to run Playwright tests with much higher parallelization across different operating system-browser combinations simultaneously. This means faster test runs with broader scenario coverage, which helps speed up delivery of features without sacrificing quality. The service also enables you to publish test results and related artifacts to the service and view them in the service portal enabling faster and easier troubleshooting. With Microsoft Playwright Testing service, you can release features faster and more confidently.
```

Comment:
nit: which cloud? Is it bound to Azure or can be used by any cloud provider? If it's an Azure service it might make sense to specify.